view/src/components/surplus.js

```diff
@@ -151,12 +172,14 @@ class Surplus extends Component {
       cost: "",
       totalQuantityAvailable: "",
       packagingType: "",
-      // page state
+      // Page state
       errors: [],
-      open: false, // use to open the edit / add dialog (form)
+      open: false, //  Used to open the edit / add dialog (form)
```

Comment:
```suggestion
      open: false,  //  Used to open the edit / add dialog (form)
```
two spaces before the beginning of the comment (here and elsewhere)